salt/metalk8s/orchestrate/deploy_node.sls

```diff
@@ -20,6 +20,26 @@ Install python36:
     - raw_shell: true
     - roster: kubernetes
 
+Set grains ssh:
+  salt.state:
+    - ssh: true
+    - roster: kubernetes
+    - tgt: {{ node_name }}
+    - saltenv: metalk8s-{{ version }}
+    - sls:
+      - metalk8s.node.grains
+    - require:
+      - metalk8s: Install python36
```

Comment:
Yes for several reasons:
- grains are not set in the same place when running ssh and salt-minion (basically when we run this state in salt-ssh it will just put grains the `thin_dir` of the "minion" and not in the `/etc/salt/grains` file as the salt-minion will)
- this `Set grains ssh` is only run if the salt-minion is not yet configured (and we may need one day to add some grains and need to run this state also during upgrade ...)